src/pages/components/tile/usage.mdx

```diff
@@ -482,96 +524,39 @@ version.
 
 - Standard layout tiles are the same in height and width as the rest of them in
```

Comment:
There are three basic layouts for tiles: standard, vertical masonry, and horizontal masonry. The standard layout will be the most commonly used version.

* 		Standard layout tiles are the same in height and width as all other tiles in the group.
* 		In a vertical masonry layout, tiles can vary in height, but are consistent in width.
* 		In a horizontal masonry layout, tiles can vary in width; different rows of tiles may vary in height, but tiles within a row should be consistent in height.